typescript/interchain/contracts/OwnableMulticall.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.13;
+
+// ============ External Imports ============
+
+import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+struct Call {
+    address to;
+    bytes data;
```

Comment:
Maybe add `value` here? default relayers don't support it but might be nice to just add?

Reply:
punting for now